spec/request/tasks_spec.rb

```diff
@@ -290,82 +398,64 @@
   end
 
   describe 'GET /v3/tasks/:guid' do
-    it 'returns a json representation of the task with the requested guid' do
-      task = VCAP::CloudController::TaskModel.make(
-        name:         'task',
-        command:      'echo task',
-        app_guid:     app_model.guid,
-        droplet:      app_model.droplet,
-        memory_in_mb: 5,
-        disk_in_mb:   50,
-      )
-      task_guid = task.guid
-
-      get "/v3/tasks/#{task_guid}", nil, developer_headers
-
-      expected_response = {
-        'guid'         => task_guid,
-        'sequence_id'  => task.sequence_id,
-        'name'         => 'task',
-        'command'      => 'echo task',
-        'state'        => 'RUNNING',
-        'memory_in_mb' => 5,
-        'disk_in_mb'   => 50,
-        'result'       => {
-          'failure_reason' => nil
-        },
-        'droplet_guid' => task.droplet.guid,
-        'relationships' => { 'app' => { 'data' => { 'guid' => app_model.guid } } },
-        'metadata' => { 'labels' => {}, 'annotations' => {} },
-        'created_at'   => iso8601,
-        'updated_at'   => iso8601,
-        'links'        => {
-          'self' => {
-            'href' => "#{link_prefix}/v3/tasks/#{task_guid}"
-          },
-          'app' => {
-            'href' => "#{link_prefix}/v3/apps/#{app_model.guid}"
-          },
-          'cancel' => {
-            'href' => "#{link_prefix}/v3/tasks/#{task_guid}/actions/cancel",
-            'method' => 'POST',
+    it_behaves_like 'permissions for single object endpoint', ['space_supporter'] do
```

Comment:
Quick question on this one, is there a reason we're only checking against the `space_supporter` role?

Reply:
Just me being a numpty who forgets to put things back the way they were after changing them for debugging ;-)

Thanks for spotting that. I've just pushed a fix so that it tests all roles again.

Reply:
Happens to the best of us :) Perfect, thanks for pushing the change so quickly!